tests/efi-vars-editor-vm

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+set -eux
+
+architecture="$(uname -m)"
+if [ "${architecture}" != "x86_64" ] && [ "${architecture}" != "aarch64" ]; then
+  echo "Skipping test as EFI is not supported on ${architecture}"
+  # shellcheck disable=SC2034
+  FAIL=0
+  exit 0
+fi
+
+# Install LXD
+install_lxd
+
+# Configure LXD
+lxc network create lxdbr0
+lxc profile device add default eth0 nic network=lxdbr0
+
+poolName="vmpool$$"
+poolDriver=dir
+
+echo "==> Create storage pool using driver ${poolDriver}"
+lxc storage create "${poolName}" "${poolDriver}"
+
+extra_cleanup() {
+  echo "==> extra_cleanup"
+
+  echo "==> Stopping and deleting VM"
+  lxc delete -f v1
+
+  lxc profile device remove default eth0
+
+  echo "==> Deleting storage pool"
+  lxc storage delete "${poolName}"
+
+  echo "==> Deleting network"
+  lxc network delete lxdbr0
+}
+
+echo "==> Create VM and boot"
+lxc launch "${TEST_IMG:-ubuntu-minimal-daily:22.04}" v1 --vm -s "${poolName}"
+waitInstanceReady v1
```

Comment:
Do we actually need for the boot to complete at this point?

Reply:
I don't think so. Have removed.